node/subsystem/src/jaeger.rs

```diff
@@ -0,0 +1,256 @@
+// Copyright 2020 Parity Technologies (UK) Ltd.
+// This file is part of Polkadot.
+
+// Polkadot is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+
+// Polkadot is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+
+// You should have received a copy of the GNU General Public License
+// along with Polkadot.  If not, see <http://www.gnu.org/licenses/>.
+
+//! Jaeger integration.
+//!
+//! See <https://www.jaegertracing.io/> for an introduction.
+//!
+//! The easiest way to try Jaeger is:
+//!
+//! - Start a docker container with the all-in-one docker image (see below).
+//! - Open your browser and navigate to <https://localhost:16686> to acces the UI.
```

Comment:
I've tried to follow this tutorial, but
* `podman run` failed with `Error: unable to pull jaegertracing/all-in-one: image name provided is a short name and no search registries are defined in the registries config file.` Do I need to provide a config?
replacing `podman` with `docker` made it work
* localhost returned `SSL_ERROR_RX_RECORD_TOO_LONG`, any hints?

Reply:
@ordian either you need to prefix it with `docker.io/jaegertracing/all-in-one` or add it explicitly